src/Formio.js

```diff
@@ -325,37 +331,7 @@ class Formio {
   }
 
   loadForm(query, opts) {
-    return this.load('form', query, opts)
```

Comment:
This looks like it will break reverse compatibility where it would no longer work if they upgrade renderer pointed to a older server. Is there a way we can do this so that it will not break reverse compatibility?  Maybe go ahead and fetch the form with the URL, and then if it returns the correct schema, then don't load the other requests?